docs/chapters/flow/index.md

```diff
@@ -529,4 +529,26 @@ Note that more than one `queryset` can be passed with any request.
 ```python
     with Flow().load_config('flow.yml') as search_flow:
         search_flow.search(input_fn=docs, output_fn=print_results, queryset=[top_k_queryset])
-```
\ No newline at end of file
+```
+
+## The Score Field
+
+In ranking document matches, Jina uses several algorithms to compute a `score` field. This field can mean either *distance* (with "smaller is better") or *similarity* (with "higher is better"). As, an example, the `NumpyIndexer` uses k-NN (with various distance metrics) to calculate a *distance*. On the other hand, the `MinRanker` uses a the function *1/(1+s)* (where *s* is the min. score from all `chunks`) in order to "bubble up" the score from child `chunks` to the parent, thus "larger is better".
+
+This can pe problematic when deciding when sorting the vector of results. Fortunately, Jina provides the name of the operator that has performed the scoring in the field `"score"`:
+
+```
+/**
+ * Represents the relevance model to `ref_id`
+ */
+message NamedScore {
+    float value = 1; // value
+    string op_name = 2; // the name of the operator/score function
+    ...
+}
+```
+
+In the above we can see `"op_name"`. As the comment suggests, this is the name of the operator that has performed the sorting. Based on this, the `"value"` field is either a *distance* ("smaller is better"), or a *similarity* ("higher is better"). 
```

Comment:
Again here, similarity is not the only option. Similarity or distance are measures mostly related to `vectors` or `objects` relation to another, but `relevance` can be something more arbitrary

Reply:
Updated